modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/css/_click-goal.scss

```diff
@@ -10,70 +10,72 @@ $target-overlay-border-color: rgb(11, 95, 255);
 // menu, and which we have to slide under while scrolling.
 $target-z-index: 900;
 
-.lfr-segments-experiment-click-goal-root {
-	bottom: 0;
-	left: 0;
-	position: absolute;
-	right: 0;
-	top: 0;
-	z-index: $target-z-index;
-}
-
-.lfr-segments-experiment-click-goal-target {
-	display: flex;
-	flex-direction: column;
-	justify-content: space-between;
-	pointer-events: none;
-	position: absolute;
-}
-
-.lfr-segments-experiment-click-goal-target-delete {
-	cursor: pointer;
-}
-
-.lfr-segments-experiment-click-goal-target-overlay {
-	background-color: $target-overlay-background-color;
-	border: 2px solid $target-overlay-border-color;
-	cursor: pointer;
-	pointer-events: auto;
-	position: relative;
-
-	&:before {
-		border: 2px solid rgba(247, 248, 249, 1);
+html#{$cadmin-selector} {
+	.lfr-segments-experiment-click-goal-root {
```

Comment:
Should wrap all this in `.cadmin {}`

Reply:
this selector apply to a part not under the cadmin selector, as described [here](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/blog/2021/05/19/1557659937/Isolating+Admin+Components+From+CSS+Customizations+for+Site+Creators), it's same case `.my-other-component-that-is-mixed-with-user-facing-content`